actions/php/codeStyle/entrypoint.sh

```
@@ -22,26 +22,32 @@ run_action() {
+        git fetch origin main:refs/remotes/origin/main
+
         # Get changed PHP files and run rector only on them
-        changed_files=$(git diff --name-only --diff-filter=ACMR origin/main...HEAD | grep '\.php$' || true)
+        # Use two-dot syntax instead of three-dot to avoid "no merge base" error
+        changed_files=$(git diff --name-only --diff-filter=ACMR origin/main..HEAD | grep '\.php$' || true)
 
         if [ -n "$changed_files" ]; then
             echo "Running Rector on changed files"
-            "$GH_DIRECTORY"/vendor/bin/rector process "$changed_files"
+            # Run rector from workspace root where the file paths are relative to
+            cd "$GITHUB_WORKSPACE"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)